src/gh/components/DF_joint_segmentator/code.py

```diff
@@ -0,0 +1,69 @@
+#! python3
+
+import Rhino
+import diffCheck
+from diffCheck import diffcheck_bindings
+from diffCheck import df_cvt_bindings as df_cvt
+import diffCheck.df_util
+import scriptcontext as sc
+
+ABSTOL = Rhino.RhinoDoc.ActiveDoc.ModelAbsoluteTolerance
+
+def main(i_clusters, i_joints, i_joint_ids):
```

Comment:
Please follow the class implementation as in the other components see example:
https://github.com/diffCheckOrg/diffCheck/blob/main/src/gh/components/DF_cloud_size_downsample/code.py